content/rest-api/filter.md

```diff
@@ -912,6 +984,7 @@ As a result you will receive the following answer:
 When requesting a [list of assets via the REST API](/api-reference.html#get_assets), you can apply filters to get only the ones you want and also the kind of information you want in them.
 
 ### By update date
+
 ::: availability versions=3.2,4.0,5.0,SaaS editions=EE
```

Comment:
Could we have 2 tables? 

The first with 
`::: availability versions=3.2,4.0,5.0,SaaS editions=EE`
the operator `>` 

and the second with 
`::: availability versions=SaaS editions=EE`
with all the operators

Reply:
![image](https://user-images.githubusercontent.com/1671213/121038798-db28fc80-c7b0-11eb-9bf4-e7ba1076adcb.png)